stdlib/public/SDK/Foundation/Data.swift

```diff
@@ -1684,20 +1684,20 @@ public struct Data : ReferenceConvertible, Equatable, Hashable, RandomAccessColl
     }
     
     public subscript(bounds: Range<Index>) -> Data {
-        @inline(__always)
+        @inlinable
```

Comment:
Just to make sure I understand this part: This is inlinable, but only for Data itself, because others outside the module can't see the backing type?

Reply:
hmm it looks like the squash pulled together the revert back to inlines. This has been going back and forth to get appropriate numbers for perf impact so I will have to re-apply the inline removal it looks.